Gruvo/Gruvo/Models/UserLoginModel.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Gruvo.Models
+{
+    public class UserLoginModel
```

Comment:
This model is used in communication layer. All these models should be separated from real models and moved to DTL (data transfer layer) folder.